src/presentation/api/router.go

```diff
@@ -15,17 +15,20 @@ type Router struct {
 	baseRoute       *echo.Group
```

Comment:
You forgot to update api.go and update swagger docs.

Reply:
Done!